R/PlantArea.R

```diff
@@ -188,7 +188,7 @@ plantAreaDensity <- function(vxsp, vx,
 #'
 #' @examples
 #' vxsp <- readVoxelSpace(system.file("extdata", "tls_sample.vox", package = "AMAPVox"))
-#' vxsp@data <- merge(vxsp@data, plantAreaDensity(vxsp), on = list(i, j, k))
+#' vxsp@data <- merge(vxsp@data, plantAreaDensity(vxsp), by = c("i", "j", "k"))
```

Comment:
@philippeverley I believe the issue is that you have only updated the roxygen comments, but you should also update the man/*.Rd files.